testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/tempto/tempto-configuration-for-hms.yaml

```diff
@@ -0,0 +1,15 @@
+hdfs:
+    webhdfs:
+        # 9870 is the name node's default port in Hadoop 3
+        uri: http://${databases.hive.host}:9870
```

Comment:
instead of defining it here, let's make the new config load both: `tempto-configuration-for-hive3` and also `tempto-configuration-for-hms-only` -- new file with just 
```
hive:
    # to make sure hive server is not used, we only want to run tests that uses HMS
    jdbc_url: jdbc:hive2://${databases.hive.host}:99999
```